packages/crypto/lib/constants.js

```diff
@@ -35,3 +35,46 @@ exports.CONFIGURATIONS = Object.freeze({
     TESTNET: configTestnet
   }
 })
+
+/**
+ * Lookup table for v1 backwards compatibility.
+ * @type {Object}
+ */
+exports.OUTLOOKTABLEFIX = Object.freeze({
```

Comment:
We should export this only for mainnet (as it is a fix on mainnet blockids). In the unlikely hood that there is a id collision in another network (well not much unlikely since this is an id on 8bytes), this would rekt that network.

I think it is better to move this to https://github.com/JeremiGendron/core/packages/crypto/lib/networks/ark/mainnet.json#L56 instead